components/function-controller/README.md

```diff
@@ -0,0 +1,176 @@
+# Function Controller
+
+The Knative Function Controller is a Kubernetes controller that enables Kyma to manage Function resources. It uses Knative Build and Knative Serving under the hood.
+
+## Prerequisites
+
+The Function Controller requires the following components to be installed:
+
+- Knative Build (v0.6.0)
+- Knative Serving (v0.6.1)
+- Istio (v1.0.7)
+
+## Set up the environment
+
+Follow these steps to prepare the environment you will use to deploy the Controller.
+
+1. Export the following environment variables:
+
+    | Variable        | Description | Sample value |
+    | --------------- | ----------- |--------------|
+    | **IMG** | The full image name the Function Controller will be tagged with. | `gcr.io/my-project/function-controller` for GCR, `my-user/function-controller` for Docker Hub |
+    | **FN_REGISTRY** | The URL of the container registry Function images will be pushed to. Used for authentication. | `https://gcr.io/` for GCR, `https://index.docker.io/v1/` for Docker Hub |
+    | **FN_REPOSITORY** | The name of the container repository Function images will be pushed to. | `gcr.io/my-project` for GCR, `my-user` for Docker Hub |
+    | **FN_NAMESPACE** | The Namespace where Functions are deployed. | |
+
+    See the example:
```

Comment:
Shouldn't we also add **FN_NAMESPACE** variable to the sample?

Reply:
IMO the user should know about namespaces but extra info should not hurt

Reply:
Good. Added then.